### ISSUE TITLE
Inquiry about the layout detection

### ISSUE THREAD

**Issue by Skyer19** [opendatalab/PDF-Extract-Kit#188]
May I know if I only need the layout detection, could I only use the model and code from [DocLayout-YOLO](https://github.com/opendatalab/DocLayout-YOLO/tree/main)? Do [PDF-Extract-Kit](https://github.com/opendatalab/PDF-Extract-Kit) add some extra functions for layout detection？

As far as I know, DocLayout-YOLO is one of the model implemented in PDF-Extract-Kit](https://github.com/opendatalab/PDF-Extract-Kit) , and can give the best results.

Thank you.

**Comment by wufan-tb** [comment]
Yes, PDF-Extract-Kit is same as DocLayout-YOLO for users who need layout detection only.
